pages/data-visualization/install-and-connect.mdx

```diff
@@ -83,18 +83,24 @@ If you encounter a security warning while installing Memgraph Lab on Windows, yo
 
 {<h3>Set up connection</h3>}
 
-After you start Memgraph Lab, you should be presented with a login
-screen.
+After starting Memgraph Lab, you will see the login screen. Follow
+these steps to connect:
 
-These are the default settings:
+1. Click on "New connection" in the sidebar.
+2. Select "Memgraph instance: Connect to a standalone instance".
+   Feel free to check [other connections types](/data-visualization/user-manual/connection-types).
 
-- Leave the `Username` and `Password` fields **empty**.
-- The `Host` field can be either **`localhost`**, **`127.0.0.1`** or
-   **`0.0.0.0`**, or change it appropriately.
-- The `Port` field should be **`7687`**. Every Memgraph instance is listening
-   on this port by default.
-- The `Encrypted` option should be **disabled** and display `SSL Off` by
-   default.
+Now, input the connection information:
+
+- **Host**: Enter `localhost`, `host.docker.internal`, `127.0.0.1`,
+  `0.0.0.0`, or modify it as needed.
+- **Port**: Default is `7687`, which Memgraph uses by default.
+- **Database name**: Leave empty unless connecting to a
+  [multi-tenant Memgraph instance](/database-management/multi-tenancy).
+- **Logs port**: Default is `7444`, the port for the web socket
+  logs.
+- **Encrypted**: Ensure this option is disabled by default.
```

Comment:
What value will enable or disable? Is that "true", "false", "0", "1", or something else?

Reply:
Regarding the value, it is in the second sentence: "Enable it if your Memgraph has SSL enabled."